lms/templates/vert_module.html

```diff
@@ -7,7 +11,7 @@
 <div class="vert-mod">
 % for idx, item in enumerate(items):
   <div class="vert vert-${idx}" data-id="${item['id']}">
-    ${item['content'] | n, unicode}
+    ${HTML(item['content'])}
```

Comment:
Is this item actually HTML? This seems like it could be risky to treat it as HTML.